client/rest-high-level/src/main/java/org/elasticsearch/client/Request.java

```diff
@@ -69,8 +71,21 @@ public String toString() {
                 '}';
     }
 
-    static Request ping() {
-        return new Request("HEAD", "/", Collections.emptyMap(), null);
+    // For developers please add your new methods in the alphabetical order
```

Comment:
I think this comment won't be enough to enforce this convention... why does order matter anyways @tlrx ?